proto/viam/app/data/v1/data.proto

```diff
@@ -141,7 +151,7 @@ message DeleteTabularDataByFilterRequest {
 
 // DeleteBinaryDataByFilterResponse returns the number of tabular datapoints deleted when a filter is provided
 message DeleteTabularDataByFilterResponse {
-  int64 deleted_count = 1;
+  uint64 deleted_count = 1;
```

Comment:
I believe this will require a corresponding app change for the current implementation

Reply:
Yup you're right, updated the app code to reflect this change